packages/eslint-config-ckeditor5/.eslintrc.js

```diff
@@ -308,7 +308,8 @@ module.exports = {
 		'yield-star-spacing': [
 			'error',
 			'after'
-		]
+		],
+		'ckeditor5-rules/prevent-license-key-leak': 'error'
```

Comment:
I lost track of which rules are oriented by CKEditor 5 and related projects and which touch only CKEditor 5.

I believe `eslint-config-ckeditor5` keeps generic rules for all CKEditor 5 projects, so this change should only be applied in the CKEditor 5 repository. OTOH, if an integration project leaks a license key, we have an issue, too. So perhaps it makes sense to disallow committing such phrases. In the worst case, we will disable this rule in a custom project.